indexer/Cargo.toml

```diff
@@ -14,8 +14,10 @@ cfg-if = "1.0"
 diesel = { version = "1.4", features = ["postgres", "serde_json"] }
 fuel-core = { version = "0.6.4" }
 fuel-gql-client = { version = "0.6.4" }
+fuel-indexer = { path = "../lib" }
```

Comment:
Could you also add the `version` here? This will be needed when publishing the crate.